pkg/utils/common-preset.go

```diff
@@ -182,3 +184,29 @@ func ConfigAdapterVolume() (corev1.Volume, corev1.VolumeMount) {
 	}
 	return volume, volumeMount
 }
+
+func ConfigWeightsVolume() (corev1.Volume, corev1.VolumeMount) {
```

Comment:
lets be explicit
ConfigModelWeightsVolume

Reply:
done